### PR TITLE
make the recovery phrase errors look good

### PR DESCRIPTION
**Pull request by meodai**
https://user-images.githubusercontent.com/608386/217879892-d5a3fa9f-34fc-48c3-8818-c224a97f6594.mov

### CODE REVIEW

File: src/frontend/src/flows/recovery/recoverWith/phrase.ts [hunk 1]
```diff
@@ -106,7 +88,7 @@ const init = (
     // time and the user may have left the page; we may effectively pick up the wrong element.
     const warningsDiv = document.getElementById("warnings") as HTMLDivElement;
     const warningsBox = document.querySelector(
-      "details[data-id=phrase-warnings]"
+      "[data-id=phrase-warnings]"
     ) as HTMLDetailsElement;
```
Comment (peterpeterparker) [inline src/frontend/src/flows/recovery/recoverWith/phrase.ts:92]: it isn't an `HTMLDetailElement` anymore

**Review by peterpeterparker — APPROVED** [review]
LGTM, a minor comment.

File: src/frontend/src/flows/recovery/recoverWith/phrase.ts (hunk 1 above)
Comment (meodai) [reply src/frontend/src/flows/recovery/recoverWith/phrase.ts:92]: oh true that thanks for catching it :-*

**Review by nmattia — APPROVED** [review]
LGTM thanks!

**Pull request merged**